packages/experimental/Badge/src/Badge.tsx

```diff
@@ -41,9 +41,17 @@ export const Badge = compose<BadgeType>({
       const { icon, iconPosition = 'before', ...mergedProps } = mergeProps(badge, final);
       return (
         <Slots.root {...mergedProps}>
-          {icon && iconPosition === 'before' && <Slots.icon {...iconProps} />}
-          {Children.map(children, (child) => (typeof child === 'string' ? <Slots.text key="text">{child}</Slots.text> : child))}
-          {icon && iconPosition === 'after' && <Slots.icon {...iconProps} />}
+          {icon && iconPosition === 'before' && <Slots.icon accessible={false} {...iconProps} />}
+          {Children.map(children, (child) =>
+            typeof child === 'string' ? (
+              <Slots.text accessible={false} key="text">
```

Comment:
Just to confirm, so we never want Badge to be accessible or receive focus? I'm wondering if setting the default accessible value to false but allowing it to be overridden makes more sense?

Reply:
I think if our partners will need this configurable we'll add it. But the spec says that Badges do not receive focus and "Badges should be given a meaningful description. This description will be applied, via “aria-describedby” to the element decorated by the Badge. For example, "Chat, 6 unread" or similar. General guidance is that the badge icon is marked as “visible” by default."

Reply:
Yeah, badges are not typically focusable or interact-able so it's up to the parent component to make sure the information conveyed by the badge is accessible to the user (I've seen such info added to the accessibilityLabel in the past).

I'm not sure that accessibilityDescribedBy would work if we passed it the ID to a badge if the badge is not accessible. I believe we try to look up the UIA provider associated with the ID for win32 and if the badge is not accessible, we won't get any useful information.

Reply:
I'll check accessibilityDescribedBy when I update the Tester for the Bagde